sdk/python/feast/errors.py

```diff
@@ -1,6 +1,17 @@
 from colorama import Fore, Style
 
 
+class DataSourceNotFoundException(Exception):
+    pass
+
+
+class BigQuerySourceNotFoundException(DataSourceNotFoundException):
```

Comment:
I'm not sure if it's a good idea to have implementation specific errors. We are moving to having all implementations be self-contained. Any downside to using `DataSourceNotFoundException` instead of subclassing?

Reply:
Ah thanks for letting me know - I was following the example of the `FeastObjectNotFoundException` subclassing pattern [here]( https://github.com/feast-dev/feast/blob/master/sdk/python/feast/errors.py#L6-L33), but since the `DataSourceNotFoundException`s are so similar (and we're not checking `FileSource` yet), I'll just change it to use `DataSourceNotFoundException` instead.